lib/blocs/list_state.dart

```diff
@@ -61,22 +72,30 @@ class ListState<T> extends Equatable {
       : message = null,
         isLoading = true,
         isLoadingMore = false,
-        isDone = true;
+        isLoadingMoreUp = false,
+        isDone = true,
+        isDoneUp = true;
 
   const ListState.loadingMore({required this.results})
```

Comment:
This constructor would probably cause rendering a loading indicator at both sides, instead of only on the side that is actually loading

Reply:
On a side note, any idea if the cubit's `more[Up]()` will still work correctly if both sides are loading? This is a situation that's likely to occur naturally when only 1 page has been loaded so far

Reply:
Uhh, I think thats a datarace, its not really harmful but we should probably prevent loading up if we are loading down